src/data/vault-landing.json

```diff
@@ -37,6 +37,14 @@
       "type": "tutorial_cards",
       "tutorialSlugs": ["vault/transform", "vault/static-secrets"]
     },
+    {
+      "type": "collection_cards",
```

Comment:
I've added `collection_cards` blocks to both `/vault` & `/waypoint` content, even though they're not present in designs. Intent here is to
1. validate that the component & related data fetching works
2. make it clear to authors that both tutorial and collection cards are available for use.